foundation_cms/static/scss/components/timely_activations.scss

```
@@ -50,16 +38,31 @@
 
       .activation-card {
         flex-direction: row;
+        height: 350px;
```

let's use `rem-cal` to use the values in `rem` instead.
```suggestion
        height: rem-calc(350);
```